test/Chainweb/Test/Pact/PactSingleChainTest.hs

```diff
@@ -619,35 +613,6 @@ assertSender00Bal bal msg hist =
         ])))
     hist
 
-newBlockRewindValidate :: IO (IORef MemPoolAccess) -> IO (SQLiteEnv, PactQueue, TestBlockDb) -> TestTree
-newBlockRewindValidate mpRefIO reqIO = testCase "newBlockRewindValidate" $ do
-  (_, q, bdb) <- reqIO
-  setOneShotMempool mpRefIO chainDataMemPool
-  cut0 <- readMVar $ _bdbCut bdb -- genesis cut
-
-  -- cut 1a
-  void $ runBlock q bdb second
-  cut1a <- readMVar $ _bdbCut bdb
-
-  -- rewind, cut 1b
-  void $ swapMVar (_bdbCut bdb) cut0
-  void $ runBlock q bdb second
-
-  -- rewind to cut 1a to trigger replay with chain data bug
-  void $ swapMVar (_bdbCut bdb) cut1a
-  void $ runBlock q bdb (secondsToTimeSpan 2)
-
-  where
-
-    chainDataMemPool = mempty {
-      mpaGetBlock = \_ _ _ _ bh -> do
-          fmap V.singleton $ buildCwCmd (sshow bh) testVersion
-              $ signSender00
-              $ setFromHeader bh
-              $ set cbRPC (mkExec' "(chain-data)")
-              $ defaultCmd
-      }
```

Comment:
This test triggers `fastForward`, it seems we need to adapt it or an alternative?

Reply:
For some reason I thought this test didn't apply anymore, let me think.

Reply:
Yeah this test is testing that `newBlock` with a parent header on another fork works. Now you can't give `newBlock` a parent header at all, so I figured it's not necessary.